wgpu-hal/src/vulkan/adapter.rs

```diff
@@ -869,7 +869,9 @@ impl super::InstanceShared {
                     .insert(vk::PhysicalDeviceImageRobustnessFeaturesEXT::default());
                 builder = builder.push_next(next);
             }
-            if capabilities.supports_extension(vk::ExtRobustness2Fn::name()) {
+            if !is_intel_igpu_outdated_for_robustness2(capabilities.properties, capabilities.driver)
+                && capabilities.supports_extension(vk::ExtRobustness2Fn::name())
```

Comment:
This won't properly tell us that robustness 2 is not in use. We need to make this decision when filling out the capabilities struct.

Reply:
Okay I think I did what you were looking for